docs/notes/notes.rst

```diff
@@ -3,29 +3,62 @@
 Information about the release lifecycle can be found
 `here <https://github.com/eProsima/Fast-DDS/blob/master/RELEASE_SUPPORT.md>`_.
 
-Version 2.6.6
+Version 2.6.7
 =============
 
+This release includes the following **features**:
+
+1. Support `Autofill port` (:ref:`automatically set a port<transport_tcp_transportDescriptor>`) for TCP Transport.
+2. Define a :ref:`super client<env_vars_ros_super_client>` by environment variable
+3. Support :ref:`TCP Discovery server<use-case-tcp-discovery-server>` CLI and environment variable
+4. Define methods (:ref:`environment variable<env_vars_builtin_transports>`,
+   :ref:`rtps layer<rtps_layer_builtin_transports>`, :ref:`xml<RTPS>`) to
+   :ref:`configure transport scenarios<transport_tcp_enabling>`
+5. :ref:`Secure discovery server<DS_security>`
+
 This release includes the following **improvements**:
 
-1. Improve validation on PID_PROPERTY_LIST deserialization.
-2. Improved CPU usage of timed events thread.
-3. Improved performance on intraprocess + data-sharing.
-4. Explicitly register type object in ContentFilteredTopicExample.
-5. Improve installer generation with documentation version fallback.
-6. Improve content filter expression parameters checks and verbosity.
+1. Log warning message upon receiver resource creation failure.
+2. Use foonathan memory manager for reducing allocations in `SharedMemManager.hpp`
+3. Simplify code in `CDRMessage`.
+4. Rerun failed tests with ctest option instead of colcon's.
+5. Add CCache to all CI jobs.
+6. Upgrade CMake minimum requirement to 3.16.3
+7. Update PR checklist template. Backports and Description
```

Comment:
```suggestion
1. Log warning message upon receiver resource creation failure.
2. Add tests for reconnection with same GUID
3. Use foonathan memory manager for reducing allocations in ``SharedMemManager.hpp``
4. Simplify code in ``CDRMessage``.
5. Rerun failed tests with ctest option instead of colcon's.
6. Several improvements on CI jobs.
7. Upgrade CMake minimum requirement to 3.16.3
8. Update PR checklist template. Backports and Description
```